plugins/module_utils/network/eos/rm_templates/bgp_global.py

```diff
@@ -125,11 +125,10 @@ def _tmplt_bgp_params(config_data):
             **config_data["bgp_params"]
         )
     elif config_data["bgp_params"].get("listen"):
-        command += " listen"
+        # from eos 4.23 , 'bgp listen limit ' is replaced by 'dynamic peer max'.
+        command = "dynamic peer max "
```

Comment:
Are we not supporting eos older version? As i have not seen branching code based on version.

Reply:
No we are not.